openlabcmd/openlabcmd/cli.py

```diff
@@ -315,7 +315,7 @@ def check(self):
             self._header_print(header)
             for plugin in plugins:
```

Comment:
-1, I prefer below style to run plugin:
```python
for p in plugins:
    # p is the plugin class, plugin is the plugin instance
    plugin = p() 
    plugin.check_begin()
    plugin.check()
    plugin.check_end()
```
And ``check_begin`` is design for some reserve operation for pre-check(such as, import env, create some fake data, or something), so we should use the original ``__init__`` to initialize the class.

Reply:
+1